Socket_Adapter/Socket_Adapter.csproj

```diff
@@ -94,11 +94,18 @@
     </ProjectReference>
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <PropertyGroup>
+    <PostBuildEvent>
+      xcopy "$(TargetDir)$(TargetFileName)"  "C:\ProgramData\BHoM\Assemblies" /Y
+      xcopy "$(TargetDir)MongoDB.Bson.dll"  "C:\ProgramData\BHoM\Assemblies" /Y
+      xcopy "$(TargetDir)MongoDB.Bson.xml"  "C:\ProgramData\BHoM\Assemblies" /Y
```

Comment:
Or should the `.xml` files be copied over too?
It's in the build folder, but there are other things there as well
@IsakNaslundBh

Reply:
no only dlls